Tests/ConcordiumSwiftSDKTests/GrpcClientTests.swift

```diff
@@ -0,0 +1,37 @@
+@testable import ConcordiumSwiftSDK
+import XCTest
+
+import GRPC
+import NIOPosix
+
+/// Temporary test for exercising the gRPC client. To run, adjust the channel target to point to a running node.
+final class GrpcClientTests: XCTestCase {
```

Comment:
Was there a reason why we have two different test classes for the Grpc client?
Both seem to be doing integration testing, so don't we want to combine them into a single test class?

Reply:
Good question. I don't remember the exact reasoning, but agree that this test seems redundant now.